.devcontainer/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # [Choice] .NET version: 7.0, 7.0-bullseye-slim, 7.0-jammy, 6.0, 6.0-bullseye-slim, 6.0-jammy, 6.0-focal
 ARG VARIANT="6.0-jammy"
-FROM mcr.microsoft.com/dotnet/sdk:${VARIANT}
+FROM mcr.microsoft.com/devcontainers/dotnet:${VARIANT}
```

Comment:
https://github.com/devcontainers/images/tree/main/src/dotnet

Built on top of `mcr.microsoft.com/dotnet/sdk` & avoids the need of using `common-utils` Feature.

Reply:
Oh, the reason I avoided using that dotnet image from the devcontainer is lack of flexibility. For example,

* I can't choose the variant on my own. It's fixed to `7.0-slim-bullseye`.
* I can't choose `common-utils:2`

Reply:
Curious, what variant are you looking for which is not currently available with `mcr.microsoft.com/devcontainers/dotnet`? 

We have a wide list of available tags listed here --> https://mcr.microsoft.com/v2/devcontainers/dotnet/tags/list

Reply:
Let me explain what I understand about the devcontainer. If I choose to use `devcontainers/dotnet:${VARIANT}`, it comes with the [pre-configured `devcontainer.json` settings](https://github.com/devcontainers/images/blob/main/src/dotnet/.devcontainer/devcontainer.json), which is pinned to `7.0-slim-bullseye`. Is that correct?

If it's not, I don't understand why I don't need to include the `common-utils` feature.

Reply:
> If I choose to use devcontainers/dotnet:${VARIANT}, it comes with the [pre-configured devcontainer.json settings](https://github.com/devcontainers/images/blob/main/src/dotnet/.devcontainer/devcontainer.json), which is pinned to 7.0-slim-bullseye

If you just take a look at the [source code for dotnet](https://github.com/devcontainers/images/tree/main/src/dotnet), then yes, it will feel like the repo only creates an image for `7.0-slim-bullseye`. However, we have a tooling for images release which updates the `VARIANT` variable and that's how it releases different .NET version for different variants.

For .NET we are currently releasing these variants - https://github.com/devcontainers/images/blob/main/src/dotnet/manifest.json#L3-L8

In your case, I understand you'd want to use `dotnet:6.0-jammy`. The PR changes here will just help you with that.

Reply:
> If it's not, I don't understand why I don't need to include the common-utils feature.

If I understand your scenario, then currently in your dev config I am assuming that you want to configure a `vscode` user with defined `uid` & `gid`. Also, install some common packages, set themes etc.

The `mcr.microsoft.com/devcontainers/dotnet:*` already does that configuration for you. If you just use that image instead, then it will avoid the need for `common-utils`.